src/org/parosproxy/paros/view/AbstractParamContainerPanel.java

```diff
@@ -457,8 +498,14 @@ public void initParam(Object obj) {
     }
 
     /**
-     * This method is to be overrided by subclass.
-     *
+     * Validates all panels, throwing an exception if there's any validation error.
+     * <p>
+     * The message of the exception can be shown in GUI components (for example, an error dialogue) as it's expected to be

```

Comment:
If the message isn't being passed in, how is this supposed to be interpreted or addressed by the dev? (Applies to all occurrences in this PR.)